### ISSUE TITLE
Script does not run on all regions when asked to

### ISSUE THREAD

**Issue by anuj-modi** [aws-samples/amazon-detective-multiaccount-scripts#2]
The documentation indicates that the script should run for all supported regions when no regions are specified. Instead, it simply does nothing. I have made any modifications to the code and I am using the latest commit on master as of the time of reporting (commit hash `d0034f6309e80644e52fafb052dfaca0d653bd2f`). Here is the output:
```
2020-04-09 14:13:48,063 - INFO - Enabling members in all available Detective regions []
2020-04-09 14:13:52,383 - INFO - Assumed session for <detective master account id>.
```

It seems like this line is the problem.

https://github.com/aws-samples/amazon-detective-multiaccount-scripts/blob/d0034f6309e80644e52fafb052dfaca0d653bd2f/enableDetective.py#L110

The output of calling it in an interpreter is as follows:
```
In [3]: s.get_available_regions("detective")                                                                                                                                                                    
Out[3]: []
```

Running similar code for another service returns a non-empty list:
```
In [4]: s.get_available_regions("guardduty")                                                                                                                                                                    
Out[4]: 
['ap-east-1',
 'ap-northeast-1',
 'ap-northeast-2',
 'ap-south-1',
 'ap-southeast-1',
 'ap-southeast-2',
 'ca-central-1',
 'eu-central-1',
 'eu-north-1',
 'eu-west-1',
 'eu-west-2',
 'eu-west-3',
 'me-south-1',
 'sa-east-1',
 'us-east-1',
 'us-east-2',
 'us-west-1',
 'us-west-2']
```

**Comment by skylight75** [comment]
Please make sure to use Boto3 v1.12.39 and Python 3.7

**Comment by anuj-modi** [comment]
I ran it locally using python 3.7 and boto3 v1.12.39. I also ran it AWS CodeBuild using the 3.8 runtime and the provided boto installation and ran into the same problem

**Comment by smsilb** [comment]
Thanks for reporting this, @anuj-modi! This issue can now be resolved by upgrading botocore to >= 1.16.18.
